templates/index.md.tmpl

```diff
@@ -15,9 +15,11 @@ The provider needs to be configured with Kubernetes credentials to be used.
 
 The following examples are available to help you use the provider:
 
+- [Bootstrapping a cluster using a GitHub repository self managing SSH keypair secret)](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples/github-self-managed-ssh-keypair)
```

Comment:
Remove this one

Reply:
This has to stay, its used to render documentation when running `make docs`